layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+                                <div>
+                                    <span class="text-primary fw-bold">Mission</span>
+                                    <ul>
+                                        {{ range .mission }}
+                                            <li class="px-2 justify-content-center text-primary">{{ . }}</li>
+                                        {{ end }}
+                                    </ul>
+                                </div>
+                                <div>
+                                    <span class="text-primary fw-bold">General Task</span>
+                                    <ul>
+                                        {{ range .generalTask }}
+                                            <li class="px-2 justify-content-center">
+                                                {{ . }}
+                                            </li>
+                                        {{ end }}
+                                    </ul>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            {{ end }}
+        </div>
+    </section>
+
+    <section class="bg-body-secondary">
+        <div class="container" style="padding: 4em 1em;">
+            <div class="slider">
+                <div class="slider-inner">
+                    {{ range .Site.Params.joinusImages }}
+                        <div class="slider-item">
+                            <div class="box" style='--bg: url("{{ absURL . }}");'></div>
+                        </div>
+                    {{ end }}
+                </div>
+                <div class="control">
+                    <button id="prev">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="1.3em" height="1.3em" viewBox="0 0 24 24">
+                                <g fill="none" fill-rule="evenodd">
+                                    <path d="M24 0v24H0V0zM12.593 23.258l-.011.002l-.071.035l-.02.004l-.014-.004l-.071-.035q-.016-.005-.024.005l-.004.01l-.017.428l.005.02l.01.013l.104.074l.015.004l.012-.004l.104-.074l.012-.016l.004-.017l-.017-.427q-.004-.016-.017-.018m.265-.113l-.013.002l-.185.093l-.01.01l-.003.011l.018.43l.005.012l.008.007l.201.093q.019.005.029-.008l.004-.014l-.034-.614q-.005-.019-.02-.022m-.715.002a.02.02 0 0 0-.027.006l-.006.014l-.034.614q.001.018.017.024l.015-.002l.201-.093l.01-.008l.004-.011l.017-.43l-.003-.012l-.01-.01z" />
+                                    <path fill="white" d="M7.94 13.06a1.5 1.5 0 0 1 0-2.12l5.656-5.658a1.5 1.5 0 1 1 2.121 2.122L11.122 12l4.596 4.596a1.5 1.5 0 1 1-2.12 2.122l-5.66-5.658Z" />
+                                </g>
+                            </svg>
+                    </button>
+                    <button id="next">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="1.3em" height="1.3em" viewBox="0 0 24 24">
```

Comment:
```suggestion
                            <svg xmlns="http://www.w3.org/2000/svg" width="40" height="40" viewBox="0 0 24 24">
```
Reduce size a littttle bit
usually logos and icons we can set fix size, afraid that they will grow too big or shrink too small at some screens